legacy/scripts/exec-backup-generation.sh

```diff
@@ -53,6 +53,16 @@ fi
 ##############################################
 
 
+set -x
+# Check if backups should be scheduled
+  if [ "$LAGOON_BACKUPS_DISABLED" = true ]; then
```

Comment:
This will never be checked, you'll need to do something like this that checks the scope of the variable to see if it is a build variable: https://github.com/uselagoon/build-deploy-tool/blob/c9bed57808d0110b59f7d63b891e08a8e208c382/legacy/build-deploy-docker-compose.sh#L705-L713

Reply:
https://github.com/uselagoon/build-deploy-tool/pull/169/commits/137dfa02de52108899834445b7ab1dfe0f63b88a